src/Controller/HomeController.php

```diff
@@ -12,8 +15,24 @@ class HomeController extends AbstractController
      */
     public function index()
     {
+        $studentList = array();
```

Comment:
Nuo PHP 5.5 galima naudoti ir trumpesnį užrašymo būdą.
```suggestion
        $studentList = [];
```